packages/mui-base/src/Popper/Popper.types.ts

```diff
@@ -124,7 +123,7 @@ export interface PopperTypeMap<
 
 export type PopperProps<
   RootComponentType extends React.ElementType = PopperTypeMap['defaultComponent'],
-> = OverrideProps<PopperTypeMap<{}, RootComponentType>, RootComponentType> & {
+> = PolymorphicProps<PopperTypeMap<{}, RootComponentType>, RootComponentType> & {
   component?: RootComponentType;
 };
```

Comment:
```suggestion
> = PolymorphicProps<PopperTypeMap<{}, RootComponentType>, RootComponentType>;
```

Reply:
this `Props` also describe `Popper` from mui-material. Since we don't remove `component` prop from that `Popper`, this component props can't be removed here. Typescript will give error on the Popper.tsx (packages/mui-material/src/Popper/Popper.tsx)